operations/iuf/workflows/slingshot_management_network_switch_updates.md

```diff
@@ -0,0 +1,31 @@
+# Perform Slingshot switch and management network switch firmware updates
+
+This section updates the Slingshot switch and management network switch firmware updates.
+
+- [1. Perform Slingshot switch firmware updates](#1-perform-slingshot-switch-firmware-updates)
+- [2. Perform management network switch firmware updates](#2-perform-management-network-switch-firmware-updates)
+
+## 1. Perform Slingshot switch firmware updates
+
+**`NOTE`** This subsection is optional and can be skipped if upgrading only CSM through IUF.
+
+Instructions to perform Slingshot switch firmware updates are provided in the "Upgrade HPE Slingshot switch firmware in a CSM environment" section of the _HPE Slingshot Installation Guide for CSM_.
+
+Once this step has completed:
+
+- Slingshot switch firmware has been updated
+
+## 2. Perform management network switch firmware updates
+
+**`NOTE`** This section is optional and can be skipped or deferred unless network configuration that requires updated firmware is being applied to the system.
+
+Management network switch firmware is shipped in the HPC Firmware Pack (HFP) product tarball.
+
+Refer to [Update Management Network Firmware](../../network/management_network/firmware/update_management_network_firmware.md) for instructions on performing the switch firmware update.
+
+**`NOTE`** The firmware on spine, leaf, and CDU switches can be updated without disruption. Air-cooled compute nodes, their BMCs, and other air-cooled devices
```

Comment:
This paragraph seems to say two conflicting things.

Reply:
I copied this paragraph from how it was written in the other section. Do you have a suggestion on how to improve the wording? Or should it be removed all together or left as is?

Reply:
I'm not sure I understand what the conflict is here.

Reply:
I will leave this as is.